src/components/dialog/content/error/ReportIssueButton.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <Button
+    @click="reportIssue"
+    :label="$t('reportIssue')"
+    severity="secondary"
+    :icon="icon"
+    :disabled="submitted"
+  >
+  </Button>
+</template>
+
+<script setup lang="ts">
+import { computed, ref, defineProps } from 'vue'
+import Button from 'primevue/button'
+import { useToast } from 'primevue/usetoast'
+import { ExecutionErrorWsMessage } from '@/types/apiTypes'
+import { useI18n } from 'vue-i18n'
+import { electronAPI } from '@/utils/envUtil'
+
+const { error } = defineProps<{
+  error: ExecutionErrorWsMessage
+}>()
+const emit = defineEmits<{
+  close: []
+}>()
+
+const { t } = useI18n()
+const toast = useToast()
+const submitting = ref(false)
+const submitted = ref(false)
+const icon = computed(
+  () => `pi ${submitting.value ? 'pi-spin pi-spinner' : 'pi-send'}`
+)
+
+const reportIssue = async () => {
+  if (submitting.value) return
+  submitting.value = true
+  try {
+    await electronAPI().sendErrorToSentry(error.exception_message, {
```

Comment:
@robinjhuang PTAL. I am not very sure if this is the correct format we are expecting in the Sentry setup.

Reply:
Slack chat from @robinjhuang 
>  Can we send the logs and system info in the report as well? Everything except the workflow for now would be good i think